cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZookeeperSetOperator.java

```diff
@@ -54,4 +102,225 @@ public static boolean needsRollingUpdate(StatefulSetDiff diff) {
         }
         return false;
     }
+
+    @Override
+    public Future<Void> maybeRollingUpdate(StatefulSet ss, Predicate<Pod> podRestart) {
+        String namespace = ss.getMetadata().getNamespace();
+        String name = ss.getMetadata().getName();
+        final int replicas = ss.getSpec().getReplicas();
+        log.debug("Considering rolling update of {}/{}", namespace, name);
+        Future<Void> f = Future.succeededFuture();
+
+
```

Comment:
Two blank lines is at least one too many. I'd argue two too many, because why does the variable `zkRoll` need to be separate from the rest?